app/code/Magento/Customer/Model/FileProcessor.php

```diff
@@ -181,7 +181,6 @@ public function saveTemporaryFile($fileId)
         );
 
         $result = $uploader->save($path);
-        unset($result['path']);
```

Comment:
It was added for security reasons: https://github.com/magento/magento2/blame/2.2-develop/app/code/Magento/Customer/Model/FileProcessor.php#L184

Please figure out another way to fix bug you faced with.